### PR TITLE
feat: Add parse json keys function

### PR DESCRIPTION
**Pull request by rem1niscence**

### CODE REVIEW

File: parse/parse.go [hunk 1]
```diff
@@ -0,0 +1,104 @@
+package utils
```
Comment (pablocampogo) [inline parse/parse.go:1]: The name of this package shouldn't be utils

Comment (pablocampogo) [reply parse/parse.go:1]: JSON parser maybe?

**Review by pablocampogo — CHANGES_REQUESTED** [review]
Also please make the linting pass

File: parse/parse.go (hunk 1 above)
Comment (rem1niscence) [reply parse/parse.go:1]: only parser is good

**Review by pablocampogo — APPROVED** [review]

**Pull request merged**